source/LumiCalReco/src/LumiCalClusterer_getCalHits.cpp

```diff
@@ -78,7 +78,9 @@ int LumiCalClustererClass::getCalHits(	EVENT::LCEvent * evt,
 
       } else {
 	arm =(*_mydecoder)( calHitIn )["barrel"]; // from 1 and 2
-	phiCell = (*_mydecoder)( calHitIn )["phi"];
+	//FIXME: add check for phi range?
+	phiCell = (*_mydecoder)( calHitIn )["phi"]; // goes from -phiMax/2 to +phiMax/2
+	if(phiCell < 0 ) phiCell += _cellPhiMax; // need to put into positive range only
```

Comment:
The problem is how LumiCalReco calculates the cell position from the LumiCal internal cellID, that has to know as well about the rotation of the LumiCal (or the phiCellID here needs to be corrected for possible rotations), otherwise the calculated position of the cluster is wrong. So from the external/dd4hep cellID based on the local position in the lumical to the internal cell id back to the global position calculated in the LumiCalReco everything needs to be consistent.